plugin/trino-hive/src/test/java/io/trino/plugin/hive/TestBackgroundHiveSplitLoader.java

```diff
@@ -1140,6 +1150,7 @@ private static BackgroundHiveSplitLoader backgroundHiveSplitLoaderOfflinePartiti
                 new NamenodeStats(),
                 new CachingDirectoryLister(new HiveConfig()),
                 directExecutor(),
+                newSingleThreadScheduledExecutor(daemonThreadsNamed("hive-timeout-%s")),
```

Comment:
this executor won't be shutdown

Reply:
noticed that, not sure how to solve though (and btw, the tests don't fail).